contracts/IVoucherHub.sol

```diff
@@ -15,6 +15,7 @@ interface IVoucherHub {
         uint256 voucherType,
         uint256 value
     );
+    event VoucherToppedUp(address indexed voucher, uint256 value, uint256 voucherExpiration);
```

Comment:
Can we use the same order in `VoucherCreated` and `VoucherToppedUp`?
Update the 1st to be
```solidity
VoucherCreated(voucherAddress, owner, voucherType, voucherExpiration, value)
```
and VoucherToppedUp to
```solidity
VoucherToppedUp(voucher, voucherExpiration, value)
```

Reply:
Agree! See:
- [Update VoucherCreated event args order](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/23/commits/b1d8ac9fdd2b0c03e2501dd8feb95f28e3c3249a)
- [Update VoucherToppedUp event args order and use short expiration wording since not ambiguous](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/23/commits/cad9bb79fbc5931ede64d3ce5e9859a3b8013f7e)